inc/ThirdParty/Themes/Divi.php

```diff
@@ -172,4 +174,24 @@ public function disable_divi_jquery_body() {
 		}
 
 	}
+
+	/**
+	 * Excludes Divi's CSS files from CSS combination
+	 *
+	 * @since 3.10
+	 *
+	 * @param array $exclude_css An array of CSS  to be excluded.
+	 *
+	 * @return array the updated array of paths
+	 */
+	public function exclude_divi_css_from_combine( $exclude_css ) {
+
+		if ( (bool) $this->options->get( 'remove_unused_css', 0 ) ) {
+			return $exclude_css;
+		}
+
+		$wp_content    = wp_parse_url( content_url( '/' ), PHP_URL_PATH );
```

Comment:
`wp_parse_url()` can return `null` when using the second parameter. We should check if `$wp_content` is not null before using it on the line after.